.gitignore

```diff
@@ -38,3 +38,6 @@ venv
 *.tfvars
 *.terraform.*
 *.tfstate*
+
+test_docker_files
```

Comment:
Good idea to `.gitignore` any files you've made for manually testing Terraform code.